tests/test_geoprocessing.py

```diff
@@ -113,13 +116,11 @@ def test_reclassify_raster_missing_pixel_value(self):
             test_value: 100,
         }
         target_nodata = -1
-        with self.assertRaises(ValueError) as cm:
+        with self.assertRaises(reclass_error) as cm:

```

Comment:
In combination with line 102 above, I'd tend to do an `from .geoprocessing import ReclassificationMissingValuesError` then `self.assertRaises(ReclassificationMissingValuesError)` rather than obscure the exception through a variable. Or just `pygeoprocessing.ReclassificationMissingValuesError`. It's one of the reasons I kinda like custom exceptions when they make sense -- you can infer their meaning from their class name.

Reply:
Yep, agreed. This was me trying to appease the `line too long` linting gnomes

Reply:
Now using `pygeoprocessing.ReclassificationMissingValuesError` straight up in https://github.com/natcap/pygeoprocessing/pull/106/commits/8509356492a7b45beaa058e8184e09d1087603a7